be/src/runtime/current_thread.h

```diff
@@ -154,10 +158,13 @@ class CurrentThread {
         return res;
     }
 
+    int64_t get_consumed_bytes() const { return _consumed_bytes; }
+
 private:
     const static int64_t BATCH_SIZE = 2 * 1024 * 1024;
 
     int64_t _cache_size = 0;
+    int64_t _consumed_bytes = 0;
```

Comment:
Maybe we need some comments to explain `_consumed_bytes` and `_try_consume_mem_size`.